tests/testthat/test-search-pv.R

```diff
@@ -128,25 +132,28 @@ test_that("We can call all the legitimate HATEOAS endpoints", {
   skip_on_cran()
 
   single_item_queries <- c(
-    "https://search.patentsview.org/api/v1/assignee/10/",
-    "https://search.patentsview.org/api/v1/cpc_group/A01B/",
-    "https://search.patentsview.org/api/v1/cpc_subgroup/G01S7:4811/",
-    "https://search.patentsview.org/api/v1/cpc_subsection/A01/",
-    "https://search.patentsview.org/api/v1/inventor/10/",
-    "https://search.patentsview.org/api/v1/nber_category/1/",
-    "https://search.patentsview.org/api/v1/nber_subcategory/11/",
+    "https://search.patentsview.org/api/v1/assignee/00000ce5-b13f-4a23-a8fb-c14409ad7b68/",
```

Comment:
I'm wondering if this repetition in the URL can be made more DRY? Maybe a small function that pastes the first part of the URL with the variable second?

Reply:
Sure, that's a good idea.  I'm wondering if it should be changed to make an ordinary query first and then use the links the API returns- so the test wouldn't break if the inventor, assignee and locations ids aren't persistent.

Reply:
I pushed [6d09a0b](https://github.com/ropensci/patentsview/pull/29/commits/6d09a0b58aa11c307beaf67918ac791c88b5adf8) to DRY out the code and query for links instead of hard coding ids that may not persist.